src/openrct2/rct12/RCT12.h

```diff
@@ -169,6 +170,25 @@ enum
     RCT12_ENTITY_FLAGS_IS_CRASHED_VEHICLE_ENTITY = 1 << 7,
 };
 
+// Only written to in RCT2, not used in OpenRCT2. All of these are elements that had to be invented in RCT1.
+enum : uint32_t
+{
+    TRACK_FLAGS_CONTAINS_VERTICAL_LOOP = (1 << 7),
+    TRACK_FLAGS_CONTAINS_INLINE_TWIST = (1 << 17),
+    TRACK_FLAGS_CONTAINS_HALF_LOOP = (1 << 18),
+    TRACK_FLAGS_CONTAINS_CORKSCREW = (1 << 19),
+    TRACK_FLAGS_CONTAINS_WATER_SPLASH = (1 << 27),
+    TRACK_FLAGS_CONTAINS_BARREL_ROLL = (1 << 29),
+    TRACK_FLAGS_CONTAINS_POWERED_LIFT = (1 << 30),
+    TRACK_FLAGS_CONTAINS_LARGE_HALF_LOOP = (1u << 31),
+};
+
+enum : uint32_t
+{
+    TRACK_FLAGS2_CONTAINS_LOG_FLUME_REVERSER = (1 << 1),
+    TRACK_FLAGS2_SIX_FLAGS_RIDE_DEPRECATED = (1u << 31) // Not used anymore.
+};
```

Comment:
This seems a bit unrelated to the commit

Reply:
I can separate it out into a commit of its own, is that okay? Or would you prefer it to be another PR?

Reply:
I think another commit is fine, another PR would be too much bureaucracy